ecl2df/compdat.py

```diff
@@ -978,7 +978,11 @@ def compdat_main(args):
     write_dframe_stdout_file(compdat_df, args.output, index=False, caller_logger=logger)
 
 
-def df(eclfiles: EclFiles, initvectors: Optional[List[str]] = None) -> pd.DataFrame:
+def df(
+    eclfiles: EclFiles,
+    initvectors: Optional[List[str]] = None,
+    zonemap_filename: Optional[str] = None,
```

Comment:
this should probably be a `dict` (?) with the actual zonemap.

Reply:
Yes, I can do that. But it would require bigger changes to the `compdat` code, since it is already looking for a file named `zones.lyr`.

Reply:
Ok, no I guess it can be solved quite easily. I will add that then.